<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>掷色子</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnStartThread">
          <property name="text">
           <string>启动线程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDiceBegin">
          <property name="text">
           <string>开  始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDiceEnd">
          <property name="text">
           <string>暂  停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStopThread">
          <property name="text">
           <string>结束线程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>清空文本</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="labPic">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/images/d0.jpg</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="LabA">
        <property name="text">
         <string>Thread状态：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
